src/content/account/privacy-policy.md

```diff
@@ -176,13 +176,13 @@ For tracking purposes, we use web beacons on our Website along with other techni
 
 We use the following types of cookies for the following purposes:
 
-**Strictly Necessary Cookies**: These cookies are essential for you to browse our Website and use its intended functionality, including accessing secure areas of our Website. These cookies cannot be opted-in or out of.
+- **Strictly Necessary Cookies**: These cookies are essential for you to browse our Website and use its intended functionality, including accessing secure areas of our Website. These cookies cannot be opted-in or out of.
 
-**Performance Cookies**: These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our Website. They help us to know which pages are the most and least popular and see how visitors move around our Website. All information these cookies collect is aggregated and therefore anonymous. If you do not allow these cookies, we will not know when you have visited our site.
+- **Performance Cookies**: These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our Website. They help us to know which pages are the most and least popular and see how visitors move around our Website. All information these cookies collect is aggregated and therefore anonymous. If you do not allow these cookies, we will not know when you have visited our site.
```

Comment:
Unless this is taken from a legal document and not possible to change, we could probably go with:
```suggestion
- **Performance Cookies**: These cookies allow us to count visits and traffic sources to measure and improve our Website's performance. They help us know which pages are the most and least popular and see how visitors move around our Website. All information these cookies collect is aggregated and, therefore, anonymous. If you do not allow these cookies, we will not know when you have visited our site.
```